helm/redis-r3-external/values.yaml

```diff
@@ -41,24 +51,38 @@ id-eq-id:
   auth:
     enabled: true
   master:
+    persistence:
+      size: 200Gi
+    resources:
+      limits:
+        cpu: 3000m
+        memory: 90Gi
+      requests:
+        cpu: 500m
+        memory: 50Gi
     readinessProbe:
       enabled: false
     livenessProbe:
       enabled: false
-    resources:
-      requests:
-        memory: #200Gi
     extraFlags:
     - "--save 60 10000"
     - "--appendonly no"
-    - "--stop-writes-on-bgsave-error no"
```

Comment:
I have seen become an issue on heavy write loads, where redis would fail over. Do you think we might be better off not having it ?

Reply:
My concern was that if writes are stopped on bgsave errors, then the pod will do what it did last week -- keep going without an obvious error, but also no way of recovering itself from error. Do you know what the upside of keeping this flag in is? If you think it's useful, I'm happy to revert this change!